pilot/helpers/files.py

```diff
@@ -1,6 +1,6 @@
 from pathlib import Path
 import os
-from typing import Optional, Union
+from typing import Optional, Union, Dict
```

Comment:
Using `Dict`, `List`, etc [is deprecated](https://peps.python.org/pep-0585/#implementation) and from 3.9 (which is the minimal version we support), `dict`, etc should be used